Packs/Base/Scripts/DBotTrainTextClassifierV2/DBotTrainTextClassifierV2.py

```diff
@@ -100,6 +100,9 @@ def get_data_with_mapped_label(data, labels_mapping, tag_field):
         else:
             if original_label in labels_mapping:
                 row[tag_field] = labels_mapping[original_label]
+            elif original_label.lower() in labels_mapping:
```

Comment:
Things to notice:
1. In case some labeled as "Spam" and some as "spam" they won't consider as same label (don't think it's critical)
2. I think that this doesn't break `exist_labels_counter, missing_labels_counter` but please try to also check this

Reply:
1. Yeah I know.. not critical
2. I added UT for that